tests/fast_tests/test_pyglet_renderer.py

```diff
@@ -1,85 +1,227 @@
 from flow.renderer.pyglet_renderer import PygletRenderer as Renderer
+import numpy as np
 import os
 import unittest
-
-os.environ['TEST_FLAG'] = 'True'
+import ctypes
 
 
 class TestPygletRenderer(unittest.TestCase):
     """Tests pyglet_renderer"""
+    def setUp(self):
```

Comment:
```suggestion

    def setUp(self):
```

Reply:
this suggestion is so the changes don't cause failures when something else is merged